lib/components/toolbar/toolbar_button.dart

```diff
@@ -1,4 +1,83 @@
 import 'package:flutter/material.dart';
+import 'package:saber/data/prefs.dart';
+
+/// toolbar button size
+enum ToolbarSize {
+  big,
+  normal,
+  small,
+}
+
+extension ToolbarSizeGet on ToolbarSize {
+
+  double getButtonSize(){
+    /// return size of toolbar button
+    switch (this){
```

Comment:
I've not had time to have a proper look at this but can you refactor to use [Dart's enhanced enums](https://dart.dev/language/enums#declaring-enhanced-enums) instead, like
```dart
enum ToolbarSize {
  big(buttonSize: 22, ...),
  normal(...),
  small(...);

  const ToolbarSize({
    this.buttonSize,
    ...,
  });

  final double buttonSize;
  // ...
}
```

Reply:
I will try to rewrite code.

Reply:
code is rewriten